frontend/src/metabase/components/Schedule/utils.tsx

```diff
@@ -74,5 +74,3 @@ export const getLongestSelectLabel = (data: SelectProps["data"]) =>
     const label = typeof option === "string" ? option : option.label || "";
     return label.length > acc.length ? label : acc;
   }, "");
-
-// HIIII
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 79]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->A comment that snuck while I was showing Sloan something<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/metabase/metabase/44255?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Heeeeeeyyyyyyy